src/cpacs_other/CTiglUIDManager.h

```diff
@@ -58,7 +58,7 @@ class CTiglUIDManager
     TIGL_EXPORT void RegisterObject(const std::string& uid, void* object, const std::type_info& typeInfo);
 
     template<typename T>
-    void RegisterObject(const std::string& uid, T& object)
+    TIGL_EXPORT void RegisterObject(const std::string& uid, T& object)
```

Comment:
Please remove this export. This is a inline method which must not be exported.

Reply:
done